### PR TITLE
Intent and abilties 101

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Renames "pheromones" to "intent", and lays out the basic mechanics.

### CODE REVIEW

File: design/src/player-actions/intent.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+[Purely informational hive mind abilities](informational-tools.md) never cost intent (even if thematically it might be defensible) in order to reduce player frustration.
+
+## Abilities
+
+Intent can be used for several abilities, each of which is [unlocked](intent.md#unlocking-and-upgrading) as the player plays the game.
+They are listed here in the approximate unlock order.
+
+Abilties are always **dual**: their effect can always be thematically mirrored.
+The more straightforward positive effect is always on the left mouse button,
+while the negative effect is on the right mouse button.
```
Comment (TimJentzsch) [inline design/src/player-actions/intent.md:40]: We should already think about gamepad controls as well :)

File: design/src/player-actions/intent.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+*Lure* gathers units to the area.
+
+*Warning* repels them from the area.
+This is more efficient to use than Lure.
+
+### *Flourish* / *Fallow*
+
+*Flourish* boosts growth and action rate of all units and buildings in the area.
+Costs fo growth and inputs to recipes are unchanged.
```
Comment (TimJentzsch) [inline design/src/player-actions/intent.md:52]:
```suggestion
Costs for growth and inputs to recipes are unchanged.
```

File: design/src/player-actions/intent.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Maintenance costs are increased proportionally.
+
+*Fallow* slows down growth and action rate of all unit and buildings in the area.
+Maintenance costs are decreased proportionally.
+Recover intent for each affected organism for the duration of the effect.
+When used in dense areas, Fallow is intent-positive (at the cost of productivity).
+It can also be used to slow down consumption in times of need and reduce generation of unwanted products without destroying the organism.
+
+## Unlocking and upgrading
```
Comment (TimJentzsch) [inline design/src/player-actions/intent.md:61]: Is intent _always_ going to be a manual player action?
Or can it also be automated in the late-game?

I'm thinking of that similar to how bots automate construction/destruction of entities in the Factorio late-game.

File: design/src/player-actions/intent.md (hunk 1 above)
Comment (alice-i-cecile) [reply design/src/player-actions/intent.md:40]: Agreed! Plan is to do a full UI mockup + keybindings soon.

File: design/src/player-actions/intent.md (hunk 3 above)
Comment (alice-i-cecile) [reply design/src/player-actions/intent.md:61]: I'm really reluctant to make this automatable: it encourages tedious usage.

**Pull request merged**